src/assets/i18n/de_DE.json

```diff
@@ -379,8 +379,8 @@
   "account.punchout.no_user.text": "Derzeit haben Sie keine {{0, translate, account.punchout.type.text}}-Punchout-Benutzer.",
   "account.punchout.oci.formatter.aria_label": "Format des Wertes",
   "account.punchout.oci.info.url.helptext": "Bitte verwenden Sie die folgende URL für die Konfiguration in Ihrem OCI-Procurement-System.  Ersetzen Sie <USERNAME> und <PASSWORD> durch die OCI Punchout-Benutzerdaten und setzen Sie die <HOOK_URL> entsprechend Ihrem Procurement-System.",
-  "account.punchout.oci.map_from.aria_label": "Wert 'map from'",
-  "account.punchout.oci.map_to.aria_label": "Wert 'map to'",
+  "account.punchout.oci.map_from.aria_label": "Wert „map from“",
+  "account.punchout.oci.map_to.aria_label": "Wert „map to“",
```

Comment:
Is this formatting of the quotation marks ok and displayed correctly? In French we also used guillemets « » there...

Reply:
This seems fine